tests/Providers/Qr/IQRCodeProviderTest.php

```diff
@@ -1,34 +1,29 @@
-<?php
+<?php declare(strict_types=1);
 
 namespace Tests\Providers\Qr;
 
 use PHPUnit\Framework\TestCase;
+use RobThree\Auth\Algorithm;
+use RobThree\Auth\Providers\Qr\HandlesDataUri;
 use RobThree\Auth\TwoFactorAuth;
 use RobThree\Auth\TwoFactorAuthException;
-use RobThree\Auth\Providers\Qr\HandlesDataUri;
 
 class IQRCodeProviderTest extends TestCase
 {
     use HandlesDataUri;
 
-    /**
-     * @return void
-     */
-    public function testTotpUriIsCorrect()
+    public function testTotpUriIsCorrect(): void
     {
         $qr = new TestQrProvider();
 
-        $tfa = new TwoFactorAuth('Test&Issuer', 6, 30, 'sha1', $qr);
+        $tfa = new TwoFactorAuth('Test&Issuer', 6, 30, Algorithm::Sha1, $qr);
```

Comment:
we can probably used named arguments and rely on defaults to reduce complexity
```suggestion
        $tfa = new TwoFactorAuth(issuer: 'Test&Issuer', qrcodeprovider: $qr);
```

Reply:
Maybe for another PR.